src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -8479,6 +8479,246 @@ def test_aks_migrate_cluster_to_cilium_dataplane(self, resource_group, resource_
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17,
+        name_prefix="clitest",
+        location="eastus",
+        preserve_default_location=True,
```

Comment:
Please remove this parameter `preserve_default_location=True,` if this case could be executed in other location.

The test sub could not create vm in this region right now.

Reply:
Thank you, @FumingZhang , removed parameter `preserve_default_location=True`.